### PR TITLE
Clear our some old TODO comments

### PR DESCRIPTION
**Pull request by jseppi**
Clears out a few old `TODO` comments from various places.

### CODE REVIEW

File: docs/monitoring.md [hunk 1]
```diff
@@ -43,10 +43,6 @@ For more details on the `/healthcheck/` endpoint, see
 CALC has a suite of tests in [production_tests](../production_tests/) for
 checking that the production site is properly configured.
 
-<!--
-TODO: Restore this information once cron jobs are supported and setup in
-      our CircleCI build.
-These tests are run as part of our daily CI cron build.
--->
+These tests are run nightly as a workflow build in CircleCI.
```
Comment (jseppi) [inline docs/monitoring.md]: Nightly tests are in, as of https://github.com/18F/calc/pull/1654 (once that gets promoted to `master`).

File: frontend/upload.py [hunk 2]
```diff
@@ -30,12 +30,6 @@ def render(self, name, value, attrs=None):
         if attrs:
             final_attrs.update(attrs)
 
-        if self.required:
-            # TODO: Django 1.10 automatically adds this attribute as needed
-            # based on the form field, so we should remove this once we
-            # upgrade.
-            final_attrs['required'] = ''
```
Comment (jseppi) [inline frontend/upload.py]: This should be taken care of as of our upgrade to Django 1.11 (https://github.com/18F/calc/pull/1640)

File: frontend/upload.py [hunk 3]
```diff
@@ -47,7 +41,7 @@ def render(self, name, value, attrs=None):
 
         nojs_preamble = ''
         if self.existing_filename:
-            if 'required' in final_attrs:
```
Comment (jseppi) [inline frontend/upload.py]: New change since you reviewed, @toolness. Still look good?

Comment (toolness) [reply frontend/upload.py]: oh sure, looks good!

**Pull request merged**